tools/Linux/kodi.sh.in

```diff
@@ -119,7 +119,10 @@ print_crash_report()
     # find in userdata dir
     single_stacktrace "$HOME" 5
     # try /proc/sys/kernel/core_pattern
-    [ -d $(dirname $(cat /proc/sys/kernel/core_pattern)) ] && single_stacktrace $(dirname $(cat /proc/sys/kernel/core_pattern)) 1
+    # Check if it does not contain a pipe to a program (see man 5 core)
+    if [[ "$(cat /proc/sys/kernel/core_pattern)" != \|* ]]; then
```

Comment:
please use single [ brackets, [[ is not posix and not supported by dash, which is ubuntus default /bin/sh